code/script.js

```diff
@@ -1,47 +1,160 @@
-// All the DOM selectors stored as short variables
+// Declare variables
+const startButton = document.getElementById('start-button');
+const startContent = document.getElementById('start-content');
+const chatBox = document.getElementById('chat-box');
 const chat = document.getElementById('chat')
+const form = document.getElementById('name-form')
+const nameInput = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper');
 
-// Global variables, if you need any, declared here
+let buttonText = "";
+let questionCounter = 0;
 
-// Functions declared here
+// Start chat by moving past start page
+startButton.addEventListener('click', () => {
+  startContent .style.display = "none";
+  chatBox.style.display = "flex";
+  questionGenerator();
+});
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// Function to move through questions
+const questionGenerator = () => {
+  questionCounter++;
+  if (questionCounter === 1) {
+    setTimeout(question1, 1000);
+  } else if (questionCounter === 2) {
+    setTimeout(question2, 1000, nameInput.value);
+  } else if (questionCounter === 3) {
+    setTimeout(question3, 1000, buttonText);
+  } else if (questionCounter === 4) {
+    setTimeout(question4, 1000);
+  } else if (questionCounter === 5) {
+    setTimeout(question5, 1000);
+  } else if (questionCounter === 6) {
+    setTimeout(endMessage, 1000, nameInput.value);
+  }
+}
+
+// Question 1
+const question1 = () => {
+  showMessage(`Hello there, what's your name?`, 'bot')
+
+  document.getElementById('name-form').onsubmit = event => {
+    event.preventDefault();
+    form.submitButton.disabled = true;
+    showMessage(nameInput.value, 'user');
+    questionGenerator();
+  }
+}
+
+// Question 2
+const question2 = (message) => {
+  
+  showMessage(`How is your day going, ${message}?`, 'bot');
+
+  inputWrapper.innerHTML = `
+    <button class="button-input">Great!</button>
+    <button class="button-input">Could be better</button>
+    `
+  addButtonListeners('button-input');
+}
+
+// Question 3
+const question3 = (answer) => {
+  console.log(buttonText);
+  if (answer === 'Great!') {
+    showMessage(`Glad to hear it! Name one thing that made you happy.`, 'bot');
+  } else {
+    showMessage(`Aw, what's bugging you?`, 'bot');
+  }
+
+  inputWrapper.innerHTML = `
+    <input id="text-input" type="text" />
+    <button id="send-btn">Send</button>
+  `
+  const textInput = document.getElementById('text-input');
+  document.getElementById('send-btn').addEventListener('click', () => {
+    showMessage(textInput.value, 'user');
+    textInput.value ='';
+    questionGenerator();
+  }, {once: true});
+}
+
+// Question 4
+const question4 = (answer) => {
```

Comment:
Seems like the answer parameter is never used here, buttonText is already defined as a global variable :)